art/metrics/privacy/membership_leakage_metrics.py

```diff
@@ -0,0 +1,89 @@
+from __future__ import absolute_import, division, print_function, unicode_literals
+from typing import TYPE_CHECKING, Optional
+
+import logging
+
+import numpy as np
+import scipy
+
+from art.utils import check_and_transform_label_format, is_probability
+from art.estimators.classification.classifier import ClassifierMixin
+
+if TYPE_CHECKING:
+    from art.estimators.classification import Classifier
+
+
+def PDTP(target_estimator: "Classifier", extra_estimator: "Classifier", x: np.ndarray, y: np.ndarray,
+         indexes: Optional[np.ndarray] = None, num_iter: Optional[int] = 10) -> np.ndarray:
+    """
+        Compute the pointwise differential training privacy metric for the given classifier and training set.
+        Taken from: https://arxiv.org/abs/1712.09136
```

Comment:
To follow the pattern in the other docstrings:
```suggestion
        | Paper link: https://arxiv.org/abs/1712.09136
```